app/src/main/java/com/pajato/android/gamechat/exp/model/Chess.java

```diff
@@ -68,6 +66,21 @@
     /** The experience icon url. */
     public String url;
 
+    // Castling Management booleans
+
+    /** Remember that the primary team queen side rook has moved */
```

Comment:
A blank line between variables will enhance readability, methinks.  Fewer variables would be even better.